playbook/actions/ai_slack.go

```diff
@@ -205,9 +206,12 @@ func slackBlocks(ctx context.Context, knowledge *KnowledgeGraph, diagnosisReport
 
 	blocks = append(blocks, divider)
 
-	if labelsSection := createSlackFieldsSection("Labels", *affectedResource.Labels); labelsSection != nil {
-		blocks = append(blocks, labelsSection)
-		blocks = append(blocks, divider)
+	if affectedResource.Labels != nil {
```

Comment:
I don't think this should be specific to AI notifications, rather their should be a generic configItem.getLabels() []Label that returns a sorted list of labels, starting with tags (filtering out common values e.g. if both `eu-west-1` and `eu-west-1a` tag values are found only return the more specific `eu-west-1a` zone, and then whitelisted labels.

getLabels should then be called by normal notifications / ai / etc.. 

For both tags and labels use custom weights:

* account
* project
* cluster
* namespace
* name
* service
* deployment
* statefulset
* daemonset
* cronjob
* pod
* job
* container
* release
* chart

then A-Z

then negative weighting (if whitelisted)
* version
* *-hash
* *generation